### PR TITLE
Changes to pass DBS validation tests and fix test of checking serverinfo API

### PR DESCRIPTION
**Pull request by vkuznet**
@yuyiguo I made separate PR to address changes required to pass DBS Validation tests. The changes reflects difference in API output and behavior between two implementations. Please review. For instance, in new implementation I added `/process` API for DBS Migration server to distinguish it from `/submit`. The former process existing request, while later only submit a request. With these set of changes I was able to pass all DBSMigration_t tests.

### CODE REVIEW

File: tests/dbsclient_t/validation/DBSValidation_t.py [hunk 1]
```diff
@@ -392,6 +402,8 @@ def check(input, output):
                     if self.debug:
                         print("wrong data-type %s, we expect a list" % type(output))
                     if input:
+                        print("input ", input)
```
Comment (yuyiguo) [inline tests/dbsclient_t/validation/DBSValidation_t.py:405]: Can you put all the prints under debugging mode? There are  a few below too.

File: tests/dbsclient_t/validation/DBSValidation_t.py [hunk 2]
```diff
@@ -417,18 +429,36 @@ def test12(self):
...
+        if self.debug:
+            print("migration_request %s" % migration_request)
+        if isinstance(migration_request, list):
+            migration_request = migration_request[0]
         self.assertTrue('migration_request_id' in migration_request['migration_details'])
+        if not migration_request['migration_details']:
+            return
         migration_request_id = migration_request['migration_details']['migration_request_id']
+        if migration_request_id == 0 or migration_request_id == '0':
+            print("there is no migration request to process")
```
Comment (yuyiguo) [inline tests/dbsclient_t/validation/DBSValidation_t.py:441]: same here for the prints

File: tests/dbsclient_t/validation/DBSValidation_t.py [hunk 3]
```diff
@@ -417,18 +429,36 @@ def test12(self):
...
+            print(migration_request)
+            return
         print("____toMigrate___")
         print(toMigrate)
         print("----------migration_request -----------")
         print(migration_request)
+        if GoServer():
+            data = {'migration_rqst_id': migration_request_id}
+            status = self.migration_api.processMigration(data)
+            print("process mig request", migration_request_id)
```
Comment (yuyiguo) [inline tests/dbsclient_t/validation/DBSValidation_t.py:451]: here too

File: tests/dbsclient_t/validation/DBSValidation_t.py [hunk 4]
```diff
@@ -507,7 +546,13 @@ def test13(self):
...
         migration_request = self.migration_api.submitMigration(toMigrate)
+        if self.debug:
+            print("migration_request %s" % migration_request)
+        if isinstance(migration_request, list):
+            migration_request = migration_request[0]
         self.assertTrue('migration_request_id' in migration_request['migration_details'])
+        if not migration_request['migration_details']:
+            return
         migration_request_id = migration_request['migration_details']['migration_request_id']
         print("____toMigrate___")
```
Comment (yuyiguo) [inline tests/dbsclient_t/validation/DBSValidation_t.py:557]: These are old prints. Now we can move them under debugging mode/

File: tests/dbsclient_t/validation/DBSValidation_t.py [hunk 5]
```diff
@@ -536,24 +584,32 @@ def check(input, output):
                                 del output[key][key2remove]
                             except KeyError:
                                 pass
+                    if key not in output:
+                        print("key %s not in %s" % (key, output))
```
Comment (yuyiguo) [inline tests/dbsclient_t/validation/DBSValidation_t.py:588]: here too

**Comment by vkuznet** [comment]
@yuyiguo I applied changes you requested (put all printous under debug mode), please review again.

**Pull request merged**